proto/src/determined/api/v1/master.proto

```diff
@@ -50,6 +50,8 @@ message GetMasterResponse {
   string branding = 9;
   // Feature flag for RBAC and user groups.
   bool rbac_enabled = 10;
+  // List of features that is on.
+  repeated string feature_switches = 11;
```

Comment:
can we please tag it as `= 12`? otherwise it'll conflict with [this SaaS change](https://github.com/determined-ai/determined-ee/blob/master/proto/src/determined/api/v1/master.proto#L62) we are yet to move to OSS.

Reply:
No problem